libraries/collide2/opcodetypes.h

```diff
@@ -24,6 +24,7 @@
 // config.h is the VS ./configure determined header.
 #include <cfloat>
 #include <cwchar>
+#include <cstdint>
```

Comment:
Why was this added?

Reply:
the file needed the C-Types (`uint_8`, etc)